cms/wagtail_hooks.py

```diff
@@ -0,0 +1,47 @@
+"""Custom hooks to configure wagtail behavior"""
+from wagtail.admin.api.views import PagesAdminAPIViewSet
+from wagtail.core import hooks
+
+COURSE_READABLE_ID_FIELD = "{prefix}coursepage__course__readable_id"
+TITLE_FIELD = "title"
+DEFAULT_ORDER = (
+    COURSE_READABLE_ID_FIELD.format(prefix=""),
+    TITLE_FIELD,
+)
+
+def parse_ordering_params(param):
+    if "ord" in param:
```

Comment:
I couldn't really get the difference b/w `ord` & `[param]` on line#22. Could you add a little description of what's happening? This function should have some docstrings.